worldengine/world.py

```diff
@@ -94,7 +85,16 @@ def _to_protobuf_matrix(matrix, p_matrix, transformation=None):
         for row in matrix:
             p_row = p_matrix.rows.add()
             for cell in row:
-                value = cell
+                '''
```

Comment:
This, too, is a little bit dirty and made necessary because of the numpy-arrays.  The only bool-map I know of is the ocean-map, but that one is used so frequently that I didn't want to leave it without numpy-support.